_data/simple-icons.json

```diff
@@ -13982,6 +13982,11 @@
             "hex": "1F36C7",
             "source": "https://www.unilever.com/our-company/the-logo"
         },
+        {
+            "title": "Uniqlo",
+            "hex": "ED1D24",
```

Comment:
```suggestion
            "hex": "FF0000",
```

Their website SVG is in primary red - we should use that too.

Reply:
https://brandpalettes.com/uniqlo-colors/ I look this website